### PR TITLE
adding ability to skip tests

### PR DESCRIPTION
**Pull request by kensipe**
based on cpu, mem, dcos version and marathon version

### CODE REVIEW

**Comment by sschneid** [comment]
@mesosphere-ci retest this please

**Comment by kensipe** [comment]
@mesosphere-ci retest this please

File: setup.py [hunk 1]
```diff
@@ -1,11 +1,10 @@
-from codecs import open as codecs_open
-from setuptools import setup, find_packages
+from setuptools import find_packages, setup
 
 
 setup(name='dcos-shakedown',
       version='1.1.15',
       description=u"DC/OS testing harness and library",
-      long_description=u"A tool and library to abstract common DC/OS-related tasks.",
+      long_description=u"A tool and library to abstract common DC/OS-related tasks.",
```
Comment (sschneid) [inline setup.py]: ?

Comment (kensipe) [inline setup.py]: oops

**Pull request merged**